backend/routes/user.js

```diff
@@ -0,0 +1,304 @@
+const express = require("express");
+const user = express.Router();
+const { PrismaClient } = require("../generated/prisma");
+const prisma = new PrismaClient();
+// create a user
+user.post("/", async (req, res) => {
+  const { id, email, password } = req.body;
+
+  try {
+    const newUser = await prisma.user.create({
+      data: {
+        email,
+        password,
+        // profile data
+        profile: {
+          create: {
+            ageRange: "",
+            gender: "",
+            occupation: "",
+          },
+        },
+        sensitiveData: {
+          create: {
+            salary: "",
+          },
+        },
+      },
+      include: {
+        profile: true,
+        sensitiveData: true,
+      },
+    });
+
+
+    return newUser;
+  } catch (error) {
+    res
+      .status(400)
+      .json({ error: "Failed to create user", details: error.message });
+    throw error;
+  }
+});
+// get a specific user's information
+user.get("/:id", async (req, res) => {
+  const { id } = req.params;
+
+  try {
+    const currentUser = await prisma.user.findUnique({
+      where: { id: parseInt(id) },
+    });
+
+    if (!user) {
```

Comment:
do you mean `currentUser`?
```suggestion
    if (!currentUser) {
```